composition-js/src/merging/merge.ts

```diff
@@ -1014,10 +1004,15 @@ class Merger {
       } else if (sourceSubgraphName === subgraphName) {
         this.errors.push(ERRORS.OVERRIDE_FROM_SELF_ERROR.err({
           message: `Source and destination subgraphs "${sourceSubgraphName}" are the same for overridden field "${coordinate}"`,
+          nodes: overrideDirective?.sourceAST,
         }));
       } else if (subgraphsWithOverride.includes(sourceSubgraphName)) {
         this.errors.push(ERRORS.OVERRIDE_SOURCE_HAS_OVERRIDE.err({
           message: `Field "${coordinate}" on subgraph "${subgraphName}" is also marked with directive @override in subgraph "${sourceSubgraphName}". Only one @override directive is allowed per field.`,
+          nodes: [
```

Comment:
nit: there is a `sourceASTs` function that I've been using to simply passing those. That is,
```typescript
nodes: sourceASTs(overrideDirective, subgraphMap[sourceSubgraphName].overrideDirective)
```
should do the trick.